_blog.yml

```
@@ -2211,3 +2211,12 @@
     - intel
     - guide
     - quantization
+
+- local: game-jam
+  title: "Announcing our first AI Game Jam 🎮"
```

To be updated if we decide to follow Dylan's suggestion.

```suggestion
  title: "Announcing the Open Source AI Game Jam 🎮"
```